### PR TITLE
First kube-up docs for Photon Controller

### PR DESCRIPTION
**Pull request by AlainRoy**
First kube-up docs for Photon Controller

1. Documentation for kube-up integration with VMware's Photon
Controller
2. Minor tweaks to vSphere documentation:
- Changed maintainer
- Added small note on vSphere parameters

### CODE REVIEW

**Comment by AlainRoy** [comment]
This can be viewed at:
http://alainroy.github.io/docs/

File: docs/getting-started-guides/photon-controller.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+Controller](http://vmware.github.io/photon-controller/)
+deployment. (Administrator access is only required for the initial
+setup: the actual creation of the cluster can be done by anyone.)
+
+2. The [Photon Controller CLI](https://github.com/vmware/photon-controller-cli)
+needs to be installed on the machine on which you'll be running kube-up. If you
+have go installed, this can be easily installed with:
+
+```shell
+go get github.com/vmware/photon-controller-cli
```
Comment (pwittrock) [inline docs/getting-started-guides/photon-controller.md]: I am getting this on my Mac.  Ideas?

>
$ go get github.com/vmware/photon-controller-cli
can't load package: package github.com/vmware/photon-controller-cli: no buildable Go source files in /Users/pwittroc/go/src/github.com/vmware/photon-controller-cli

Comment (AlainRoy) [inline docs/getting-started-guides/photon-controller.md]: Oops, my bad. It should be `go get github.com/vmware/photon-controller-cli/photon`. I've updated the pull request to reflect this. Thanks for catching this!

File: docs/getting-started-guides/index.md [hunk 2]
```diff
@@ -130,7 +131,8 @@ Vagrant              | CoreOS       | CoreOS | flannel     | [docs](/docs/gettin
 Bare-metal (Offline) | CoreOS       | CoreOS | flannel     | [docs](/docs/getting-started-guides/coreos/bare_metal_offline)              |          | Community ([@jeffbean](https://github.com/jeffbean))
 Bare-metal           | CoreOS       | CoreOS | Calico      | [docs](/docs/getting-started-guides/coreos/bare_metal_calico)               |          | Community ([@caseydavenport](https://github.com/caseydavenport))
 CloudStack           | Ansible      | CoreOS | flannel     | [docs](/docs/getting-started-guides/cloudstack)                             |          | Community ([@runseb](https://github.com/runseb))
-Vmware               |              | Debian | OVS         | [docs](/docs/getting-started-guides/vsphere)                                |          | Community ([@pietern](https://github.com/pietern))
+Vmware vSphere       |              | Debian | OVS         | [docs](/docs/getting-started-guides/vsphere)                                |          | Community ([@imkin](https://github.com/imkin))
```
Comment (johndmulhausen) [inline docs/getting-started-guides/index.md]: Can you also update the table at the bottom of vsphere.md?

File: docs/getting-started-guides/index.md [hunk 3]
```diff
@@ -130,7 +131,8 @@ Vagrant              | CoreOS       | CoreOS | flannel     | [docs](/docs/gettin
...
+Vmware Photon        |              | Debian | OVS         | [docs](/docs/getting-started-guides/photon-controller)                      |          | Community ([@alainroy](https://github.com/alainroy))
```
Comment (johndmulhausen) [inline docs/getting-started-guides/index.md]: Add a support table to the bottom of photon-controller.md a la vsphere.md that duplicates this info

File: docs/getting-started-guides/photon-controller.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+Once you have successfully reached this point, your Kubernetes cluster
+works just like any other.
+
+Note that kube-up created a Kubernetes configuration file for you in
+`~/.kube/config`. This file will allow you to use the `kubectl`
+command. It contains the IP address of the Kubernetes master as well
+as the password for the `admin` user. If you wish to use the
+Kubernetes web-based user interface you will need this password. In
+the config file you'll see a section that look like the following: you
+use hte password there. (Note that the output has been trimmed: the
```
Comment (johndmulhausen) [inline docs/getting-started-guides/photon-controller.md]: s/hte/the

File: docs/getting-started-guides/photon-controller.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+```yaml
+- name: photon-kubernetes
+  user:
+    client-certificate-data: Q2Vyd...
+    client-key-data: LS0tL...
+    password: PASSWORD-HERE
+    username: admin
+```
+
+### Removing your Kubernetes cluster
```
Comment (johndmulhausen) [inline docs/getting-started-guides/photon-controller.md]: Blank line after headings

File: docs/getting-started-guides/photon-controller.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+`kube-down` command:
+
+```shell
+KUBERNETES_PROVIDER=photon-controller cluster/kube-down.sh
+```
+
+Your Kubernetes cluster is just a set of VMs: you can manually remove
+them if you need to.
+
+### Making services publicly accessible
```
Comment (johndmulhausen) [inline docs/getting-started-guides/photon-controller.md]: Blank line after headings

File: docs/getting-started-guides/index.md (hunk 2 above)
Comment (AlainRoy) [inline docs/getting-started-guides/index.md]: I don't see a table at the bottom of vsphere.md, so I'm not sure what you're referring to. I looked in a few other getting started guides and didn't see a support table. Do you have an example of what you'd like?

File: docs/getting-started-guides/photon-controller.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+* A tenant, with associated resource ticket
+* A project within that tenant
+* VM and disk flavors, to describe the VM characteristics
+* An image: we'll use the one above
+
+When you do this, you'll need to configure the
+`cluster/photon-controller/config-common.sh` file with the names of
+the tenant, project, flavors, and image.
+
+If you prefer, you can  use the provided `cluster/photon-controller/setup-prereqs.sh`
```
Comment (pwittrock) [inline docs/getting-started-guides/photon-controller.md]: The name of the script is wrong.  Should be:

>
cluster/photon-controller/setup-prereq.sh

File: docs/getting-started-guides/photon-controller.md [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+The example below creates a Kubernetes cluster using VMware's Photon
+Controller. The cluster will have one Kubernetes master and three
+Kubernetes nodes.
+
+* TOC
+{:toc}
+
+### Prerequisites
+
+1. You need administrator access to a [VMware Photon
```
Comment (pwittrock) [inline docs/getting-started-guides/photon-controller.md]: How do I get this?  Is this outside the scope of the doc?  I am trying to test that performing the steps in this doc will get me a cluster with kbuectl setup, but don't know enough about Photo to set it up.

Comment (AlainRoy) [inline docs/getting-started-guides/photon-controller.md]: (I made the changes you requested, other than the table--I need clarification on that.)

I think it's outside the scope of this document: it's a big piece of software to install, and the installation is non-trivial. 

Elsewhere there is a link to the github page for Photon Controller. It has some documentation as well as downloads for releases. I have a movie of the kube-up process that I made for internal purposes: 
https://www.youtube.com/watch?v=NhRdONwV0U0

File: docs/getting-started-guides/index.md (hunk 2 above)
Comment (johndmulhausen) [inline docs/getting-started-guides/index.md]: @AlainRoy There should be a table duplicating the info on the page like this: http://kubernetes.io/docs/getting-started-guides/coreos/azure/#support-level

File: docs/getting-started-guides/index.md (hunk 3 above)
Comment (johndmulhausen) [inline docs/getting-started-guides/index.md]: @AlainRoy like this: http://kubernetes.io/docs/getting-started-guides/coreos/azure/#support-level

**Comment by AlainRoy** [comment]
Thanks, I've updated the docs to include the support table.

**Comment by pwittrock** [comment]
@johndmulhausen @AlainRoy Are there still open issues here?

**Comment by AlainRoy** [comment]
Not that I'm aware of--I think I've addressed all the comments.

**Pull request merged**